.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-  - "4.2"
-  - "5.4"
+  - "12"
```

Comment:
Good update! Can we update to v14 since that is currently LTS?

Reply:
Fixed in https://github.com/nteract/kernelspecs/pull/35/commits/ce9df17c53a0bfe9b92cef3822f3e35d57159134